recipe/activate.sh

```diff
@@ -21,7 +21,7 @@ echo "linker = \"${CONDA_PREFIX}/bin/x86_64-apple-darwin13.4.0-clang\"" >> ${CAR
 echo "[target.i686-unknown-linux-gnu]" >> ${CARGO_CONFIG}
 echo "linker = \"${CONDA_PREFIX}/bin/i686-conda_cos6-linux-gnu-cc\"" >> ${CARGO_CONFIG}
 echo "[target.x86_64-unknown-linux-gnu]" >> ${CARGO_CONFIG}
-echo "linker = \"${CONDA_PREFIX}/bin/x86_64-conda_cos6-linux-gnu-cc\"" >> ${CARGO_CONFIG}
+echo "linker = \"${CONDA_PREFIX}/bin/x86_64-conda_cos7-linux-gnu-cc\"" >> ${CARGO_CONFIG}
```

Comment:
I think this change is not necessary, nor wise.  Old version of gcc where using conda_cos6 as vendor name.  By change it here, we make those all compilers incompatible.  New compilers are using actually conda as vendor-name (just on ppc64le we still should stick additionally to a conda_cos7 name ... caused by cdt)

Reply:
The test fails when reverting to cos6: https://cloud.prefect.io/anaconda/flow-run/93ba490a-fe0b-4db8-8456-7c680a32bfaa?logs=&logId=46723c42-175d-4f4d-954b-562e2d9848db